api-put-object.go

```diff
@@ -55,7 +55,7 @@ func (opts *PutObjectOptions) getNumThreads() (numThreads int) {
 
 // getMetaData - constructs the headers from metadata entered by user in
 // PutObjectOptions struct
-func (opts *PutObjectOptions) getMetadata() (headers http.Header) {
+func (opts PutObjectOptions) getMetadata() (headers http.Header) {
```

Comment:
While we are at it let's rename this function to simply `Header()`

Reply:
done.